src/main/java/com/minelittlepony/PonySettingPanel.java

```diff
@@ -1,8 +1,6 @@
-package com.minelittlepony.gui;
+package com.minelittlepony;
```

Comment:
Keep this in the gui package.

Reply:
Don't you find it weird having a package for just one class?